velox/dwio/parquet/writer/Writer.h

```diff
@@ -43,7 +45,7 @@ class DefaultFlushPolicy : public dwio::common::FlushPolicy {
 
   bool shouldFlush(
       const dwio::common::StripeProgress& stripeProgress) override {
-    return stripeProgress.stripeRowCount >= rowsInRowGroup_ ||
+    return stripeProgress.stripeRowCount == rowsInRowGroup_ ||
         stripeProgress.stripeSizeEstimate >= bytesInRowGroup_;
```

Comment:
Just out of curiosity: will `stripeProgress.stripeSizeEstimate >= bytesInRowGroup_` cause similar issue?